lib/checks/aria/aria-hidden-focus.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "aria-hidden-focus",
```

Comment:
That is a good point about the scope of the check vs. the rule. However, the average user might not understand the distinction between those two messages. 

If we're calling out `disabled` as distinct from `tabindex="-1"` for focusability, it's only really applicable to form controls. Could we expose a message about `disabled` if the element is a form control? If it isn't one of those, then we could have the other two messages. I think it would be much less confusing.

- Focusable content should be removed from the DOM
- Focusable content should have tabindex="-1"

Reply:
I am going with the below check names

- `aria-hidden-focus-disable`
- `aria-hidden-focus-taborder`

Reply:
Once, I started doing the changes for splitting this to two checks, I realised this may be overkill, as we will be running `.querySelectorAll` twice for the same rule, with the only benefit being providing a nicer help text.